src/nuget/MSBuild.nuspec

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata minClientVersion="2.8.1">
+    <id>MSBuild</id>
+    <version>$version$</version>
+    <title>MSBuild APIs</title>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>
+    <projectUrl>http://go.microsoft.com/fwlink/?LinkId=624683</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <summary>MSBuild APIs</summary>
+    <description>
+      This package includes APIs that support writing MSBuild tasks.
+    </description>
+    <copyright>Copyright © Microsoft Corporation</copyright>
+    <dependencies>
+      <group targetFramework="net46">
+        <dependency id="Microsoft.Build.Framework" version="$version$" />
+        <dependency id="Microsoft.Build" version="$version$" />
+        <dependency id="Microsoft.Build.Utilities.Core" version="$version$" />
+        <dependency id="Microsoft.Build.Tasks.Core" version="$version$" />
+      </group>
+    </dependencies>
+  </metadata>
+  <files>
+    <!--<file src="_._" target="lib\net46" />-->
```

Comment:
IIRC we had to add an empty file with this name to force NuGet to create an empty folder, which we needed to convince . . . something . . . that this package targeted 4.6. So this might be needed here too.